operator/controller/src/test/java/io/apicurio/registry/operator/it/SmokeITTest.java

```diff
@@ -148,4 +166,104 @@ void testIngress() {
             return true;
         });
     }
+
+    @Test
+    void testEmptyHostDisablesIngress() {
+        // spotless:off
+        var registry = new ApicurioRegistry3();
```

Comment:
nitpick: my very personal preference is to keep those resources as `yaml` (I think it's nicer to read) but I don't feel strongly about it.

Reply:
Sure, I can do that.